### PR TITLE
feat(landing-page): open source redesign

### PR DESCRIPTION
**Pull request by AmNat789**
## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/etournity/etournity/#contributing) doc
- [ ] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

Created New Landing Page

### CODE REVIEW

**Comment by AmNat789** [comment]
Do not merge until new loading animation is added
(Still needs to be designed)

File: packages/client/utils/useVideosLoaded.ts
Comment (FLuplow) [inline packages/client/utils/useVideosLoaded.ts:1]: This seems to be unused. Are there plans to use this later?

**Review by FLuplow — CHANGES_REQUESTED** [review]
I really like the approach of extracting all the segments into their own files.

Overall relly good, but there are a few things that still should be thought over.

Do we really need to use .mp4's in the header and for the twitch clips? I think .gif would be a better format since we don't use sound there.

Our convention for size measurements is `rem`, so it would be nice to change all `px` values over to `rem`.

File: packages/client/components/landingPage/header.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          loop
+          preload="auto"
+          className={styles.video}
+          style={{ display: vidIndex === 1 ? 'block' : 'none' }}
+        >
+          <source
+            type="video/mp4"
+            src={`https://drive.google.com/uc?export=download&id=${loopingVideoUrl}`}
+          />
+        </video>
```
[the remark this answers is not in the document]
Comment (AmNat789) [reply packages/client/components/landingPage/header.tsx:48]: I think 2 videos should be used, because the second video needs to be preloaded

**Review by FLuplow — APPROVED** [review]

**Pull request merged**